aphrodite/modeling/sampling_metadata.py

```diff
@@ -526,6 +538,10 @@ def from_sampling_metadata(
             xtc_thresholds += [params.xtc_threshold] * n_seqs
             xtc_probabilities += [params.xtc_probability] * n_seqs
             nsigmas += [params.nsigma] * n_seqs
+            dry_multipliers += [params.dry_multiplier] * n_seqs
+            dry_bases += [params.dry_base] * n_seqs
+            dry_allowed_lengths += [params.dry_allowed_length] * n_seqs
+            dry_sequence_breaker_ids += [params.dry_sequence_breaker_ids] * n_seqs
```

Comment:
This feels incorrect and the likely source of the crash, but it's not clear how to correctly handle dry_sequence_breaker_ids